hypershiftoperator/deploy/helm/Chart.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: v2
+description: A Helm chart to install the Hypershift Operator and deps for ARO
+name: aor-hcp-hypershift-operator
```

Comment:
typo

Reply:
ah yes, the lesser known AOR service :) 
fixed